Tasks/AzureTestPlanV0/runTestPlan.ts

```diff
@@ -10,12 +10,21 @@ export async function run() {
 
     const testPlanInfo = await getTestPlanData();
 
+    let manualTestFlowReturnCode = 0;
+    let automatedTestFlowReturnCode = 0;
+
     // trigger manual, automated or both tests based on user's input
     if (testSelectorInput.includes('manualTests')) {
-        await manualTestsFlow(testPlanInfo);
+        manualTestFlowReturnCode = await manualTestsFlow(testPlanInfo);
     }
     if (testSelectorInput.includes('automatedTests')) {
-        await automatedTestsFlow(testPlanInfo, testSelectorInput);
+        automatedTestFlowReturnCode = await automatedTestsFlow(testPlanInfo, testSelectorInput);
+    }
+
+    if( manualTestFlowReturnCode || automatedTestFlowReturnCode){
+        tl.debug(`Execution Status Code for Manual Test Flow is ${manualTestFlowReturnCode}`);
```

Comment:
we should print these debug prints only if the corresponding option is selected, for instance, we cannot print this if manual flow is not selected

Reply:
Updated